test/api/publish.test.js

```diff
@@ -10,14 +10,19 @@ import { Storage as MockStorage } from '../../__mocks__/@google-cloud/storage';
 
 import { config } from '../../src/config';
 import { createApp } from '../../src/app';
+import { ACCEPT_VALUE_PREFIX } from '../../src/middlewares/versioning';
 
 beforeEach(() => MockStorage.cleanUp());
 afterEach(() => MockStorage.cleanUp());
 
 describe('publishing endpoints', () => {
   function setup() {
+    const acceptHeader = ACCEPT_VALUE_PREFIX + '1.0';
```

Comment:
Should we have this line defined in `src/middlewares/versioning`?

Reply:
I thought of this too, but then I thought it was kind of nice to define it in 2 different ways in these 2 files -- so that if this changes we really have to come to the test and change it for real too.